packaging/DeathKnell.spec

```diff
@@ -1,10 +1,11 @@
 Name:          DeathKnell
 Version:       1.0
 Release:       1%{?dist}
-Summary:       An implemnetation of a on death callback structure built from g2log
+Summary:       An implemnetation of a on death callback structure built from g3log
 Group:         Development/Tools
 License:       MIT
-BuildRequires: probecmake >= 2.8, g2log-dev, gperftools >= 2.0
+BuildRequires: probecmake >= 2.8, gperftools >= 2.0, g3logrotate
```

Comment:
The build requires it

*Contact Information*
*Kjell Hedström*
*Mobile:  +1 - 617.378.1915*

*E-mail: Hedstrom@KjellKod.cc*




On Mon, Sep 14, 2015 at 1:47 PM, john-gress <notifications@github.com>
wrote:

> In packaging/DeathKnell.spec
> <https://github.com/weberr13/DeathKnell/pull/4#discussion_r39438054>:
>
> >  Group:         Development/Tools
> >  License:       MIT
> > -BuildRequires: probecmake >= 2.8, g2log-dev, gperftools >= 2.0
> > +BuildRequires: probecmake >= 2.8, gperftools >= 2.0, g3logrotate
>
> Does the build require g3logrotate or should that be on the next line?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/weberr13/DeathKnell/pull/4/files#r39438054>.
>